stistools/mktrace.py

```diff
@@ -406,14 +406,32 @@ def gFitTrace(self, specimage, y1, y2):
         """
 
         sizex, sizey = specimage.shape
-        smoytrace = np.zeros(sizey).astype(np.float64)
-        boxcar_kernel = signal.boxcar(3) / 3.0
+        smoytrace = np.zeros(sizey, dtype=float)
+        boxcar_kernel = boxcar(3) / 3.0
+        fitter = fitting.LevMarLSQFitter()
+
+        # Bias 1D Gaussian fit toward the center of the range:
+        window = hann(specimage.shape[0], sym=True)
+        window /= window.sum()
```

Comment:
Could this cause problems for E1 data?

Reply:
I think it should be agnostic to Y-position unless it falls off the detector/subarray dimensions.

I did find some bad cases caused by the weights on the edges being zero, so changed the windowing slightly in the next commit.

Reply:
Oh, I think I understand...

The window it's fitting each column in is 41 pixels tall (as set by the `subim_size = 40` parameter).